### ISSUE TITLE
`disallowedSuperglobals` with `allowIn` doesn't seem to work when referencing a trait

### ISSUE THREAD

**Issue by ruudk** [spaze/phpstan-disallowed-calls#106]
I'm trying out the new [disallowedSuperglobals rule](https://github.com/spaze/phpstan-disallowed-calls/pull/105) while allowing a use inside trait:
```neon
parameters:
    disallowedSuperglobals:
        -
            superglobal: '$_SERVER'
            message: 'use <fg=yellow>Symfony\Component\HttpFoundation\Request</> object instead'
            allowIn:
                - **/src/Infrastructure/Persistence/Doctrine/DBAL/QueryBuffering.php
                - **/src/Kernel.php
```

It seems that [the fix for traits](https://github.com/spaze/phpstan-disallowed-calls/pull/103) doesn't work for super globals. 

I get this error:

```
Using $_SERVER is forbidden, use Symfony\Component\HttpFoundation\Request object instead
↳ src/Core/SearchBundle/CliCommand/IndexUsersCommand.php:24
↳ src/Infrastructure/Persistence/.../DBAL/QueryBuffering.php:24
```

/cc @ekisu @spaze

**Comment by spaze** [comment]
Oh, nice catch! Seems the code with the fix is not even called in the superglobals helper, dooooh. Guess this calls for a helper helper 🤪 Will see if the helpers can be refactored somehow.

**Comment by spaze** [comment]
Thanks again for catching the problem. I've moved a few bits and bytes there and back (mostly there tho) and released [2.3.1](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.3.1), please let me know if that works for you - it should _buuuuuut_... :-)

**Comment by ruudk** [comment]
@spaze Just tested it and it works. Thanks for the fast fix ⚡ 💙
